### ISSUE TITLE
use of #?# selector in sevecal cases without need for it 

### ISSUE THREAD

**Issue by wmyrda** [easylist/easylist#1575]
The rules below do not need the use of `#?#` selector as they are not followed by any adblock specific pattern containing `:-abp-*`.

[L8744](https://github.com/easylist/easylist/blob/master/easylist/easylist_specific_hide.txt#L8744) aol.co.uk,metabomb.net,moviefone.com#?#.trc-content-sponsored
[L8775](https://github.com/easylist/easylist/blob/master/easylist/easylist_specific_hide.txt#L8775) cesoirtv.com#?#a[onmousedown^="this.href='https://paid.outbrain.com/network/redir?"][target="_blank"]
[L10415](https://github.com/easylist/easylist/blob/master/easylist/easylist_specific_hide.txt#L10415)  webfail.com#?#div[style="width:300px;height:auto;margin:30px 0;"] + div

This causes [problems with adblock2privoxy](https://github.com/essandess/adblock2privoxy/issues/14) converter which at the moment is not able to translate Adblock specific rules. Hence those rules would easily work with regular use of element hiding syntax changing those lines into `##` would be appreciated.

**Comment by smed79** [comment]
I don't think [corossposting](https://github.com/easylist/easylistgermany/issues/31) is helpful...

> The rules below do not need the use of `#?#` selector

See https://issues.adblockplus.org/ticket/242#comment:39

**Comment by wmyrda** [comment]
Crossposting? I wrote about different rules in theirs respective repos. How is that wrong?

As for Adblockplus I am surprised to hear it there is such a issue as there is thousands of rules containing regular ##. You mean to tell mean none of them works on Chrome?

**Comment by ryanbr** [comment]
If another script is trying to parse Easylist, the onus is on the maintainers of the script to work around issues that aren't compatible.

**Comment by wmyrda** [comment]
I resolved the issue for myself long ago and conversion is easy using `sed`. I wrote this bug report for others so they do not need such a workaround. 
That being said IMHO your statement in no way disregards what this bug is about which is that according to [Adblock](https://adblockplus.org/filter-cheatsheet#elementhideemulation) this rule should be should be used only with connection with adblock specific rules. Having lists with records that do not follow documented patterns should be described somewhere as an exception. It is not, hence the bug reports to fix it.
